www/updateUes.php

```diff
@@ -36,91 +57,217 @@
     $perunUserId = $claims['data']['perunUserId'];
     $id = $claims['id'];
 } catch (Exception $ex) {
-    Logger::error('Perun.updateUes: An error occurred when the token was verifying.');
+    Logger::error(CLASS_PREFIX . 'An error occurred when the token was verifying.');
```

Comment:
Fix the message. When I read `when the token was verifying`, I get the feeling that the token has been verifying something.